From 3cbe6a034977d6da5313f2d80a0e8a30b10bdab4 Mon Sep 17 00:00:00 2001
From: "fugang.duan" <fugang.duan@nxp.com>
Date: Thu, 24 May 2018 20:32:58 +0800
Subject: [PATCH 18/23] CORE: wlan_hdd_cfg80211: disable scheduled scan if no
 PNO support

Disable scheduled scan if firmware doesn't have PNO support.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
---
 CORE/HDD/src/wlan_hdd_cfg80211.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CORE/HDD/src/wlan_hdd_cfg80211.c b/CORE/HDD/src/wlan_hdd_cfg80211.c
index b1d2647..18b6aa5 100644
--- a/CORE/HDD/src/wlan_hdd_cfg80211.c
+++ b/CORE/HDD/src/wlan_hdd_cfg80211.c
@@ -15860,7 +15860,7 @@ void wlan_hdd_cfg80211_update_wiphy_caps(struct wiphy *wiphy)
     * control comes here. Here just we need to clear it if firmware doesn't
     * have PNO support. */
    if (!pCfg->PnoOffload) {
-       wiphy->max_sched_scan_reqs = 1;
+       wiphy->max_sched_scan_reqs = 0;
        wiphy->max_sched_scan_ssids = 0;
        wiphy->max_match_sets = 0;
        wiphy->max_sched_scan_ie_len = 0;
-- 
1.9.1

